src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1014,7 +1011,7 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: DataSource,
       }
     }
   }
-
+  
```

Comment:
Do we need to roll back the transaction when we fail to abort ALL workflows? Or do we not care at that point?